incubator/cni-genie/values.yaml

```diff
@@ -0,0 +1,67 @@
+# ServiceToken: ""
+
+# By default, complete genie package will be installed including
+# multi networking support (along with admission controller) and network policy controller
+# In case network policy controller is not needed, set 'includePolicy' flag to false
+includePolicy: true
+
+confPath:
+  # Path for genie bin file
+  cniBinDirPath: /opt/cni/bin
+
+  # Path for genie conf file
+  cniConfDirPath: /etc/cni/net.d
+
+  # Kubeconfig file for CNIGenie CNI plugin
+  kubeconfigPath: /etc/cni/net.d/genie-kubeconfig
+
+# Port at which admission controller listens
+admissionControllerContainerPort: 8000
+
+# Loglevel for genie logs, logs will be part of /var/log/syslog file
+genieLogLevel: info
+
+# Labels for genie plugin and policy features
+geniePluginLabel: genie
+geniePolicyLabel: genie-policy
+genieadmissioncontrollerLabel: genie-admission-controller
+# serviceAccountNames for genie plugin and policy
+geniePluginServiceAccountName: genie-plugin
+geniePolicyServiceAccountName: genie-policy
+
+ServiceAccounts:
+  plugin:
+    # Specifies whether a service account should be created
+    create: true
+    # The name of the service account to use.
+    # If not set and create is true, a name is generated using the fullname template
+    name:
+  policy:
+    create: true
+    name:
+
+# Names for genie plugin and policy
+plugin:
+  name: genie-plugin
+  image:
+    pullpolicy: IfNotPresent
+    tag: latest
```

Comment:
Can you use image tags for stable versions. See https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md#configuration

It idea is to have repeatable releases. If one deploys this chart into QA and then into production the exact same images should be used. `latest` can move which breaks the repeatability.